### ISSUE TITLE
Allow users to specify their timezone and use it for drops

### ISSUE THREAD

**Issue by dkapell** [neinteractiveliterature/intercode#1520]
1) add timezone to account (not profile)
2) display users current time on profile page ( for contact reasons)
3) have liquid drops convert to users time for listing price changes, signups, etc
4) add a liquid drop for users timezone, to be used with the above

**Comment by nbudin** [comment]
I had thought about doing this when originally designing the UserConProfile and Convention models, but I decided against it because I think it would be confusing.

Basically: there are places in the site where we clearly want to display times in the convention's timezone (such as the schedule and events pages).  Given that, we're left with two options.  We either A) display some times in the convention timezone and some times in the user's timezone, or B) display all times in the convention timezone.

If we were to do (A), which I think is what you're requesting, it seems like it would be very important to clearly indicate to the user which timezone we're using everywhere.  I don't know that it's possible to do that across the board, considering how many UIs on the site display times.  It also seems like if we did have to do that, it doesn't really put us in a better spot than with (B), where we have to indicate the timezone in the places you're suggesting.

In summary: I think this would make the user experience worse.

**Comment by marleighnorton** [comment]
This should perhaps be bumped in priority, seeing how a lot of things are going on line and there is no sense of the one true time zone.  In that case, I'd recommend adding a timezone setting for each user's profile and always showing times in their local time.

**Comment by dkapell** [comment]
I think we'd want a convention-level setting for 'display time to attendees/users in their local timezone' if we were going to apply it to all places where time is displayed.  For online-only events that makes sense, for in-person events, we would want to only do that for things that aren't the convention schedule

**Comment by marleighnorton** [comment]
Somewhat related to #3430

**Comment by marleighnorton** [comment]
For this, I'd be inclined to keep things in con time, but add an addendum in local time in particular cases.  7pm EST (4pm PST) or some such.  So it's always clear what you're talking about.
